rest/src/server/bootstrapper.js

```diff
@@ -57,7 +58,8 @@ const createCrossDomainHeaderAdder = crossDomainConfig => (req, res) => {
 		if ('*' !== crossDomainResponseHeaders['Access-Control-Allow-Origin'])
 			crossDomainResponseHeaders.Vary = 'Origin';
 
-		Object.keys(crossDomainResponseHeaders).forEach(header => res.header(header, crossDomainResponseHeaders[header]));
+		Object.keys(crossDomainResponseHeaders).forEach(header =>
```

Comment:
?

Reply:
It looks like a code style change that is lint compatible. If we set up prettier in rest, prettier will unify any of those changes.

Reply:
Reverted code style change